### PR TITLE
Migrate helloworld in cs to 0.4.1

### PR DESCRIPTION
**Pull request by YueLiu-jina**
Part of #116 , ready for review

### CODE REVIEW

File: helloworld-in-cs/client.py [hunk 1]
```diff
@@ -24,11 +26,12 @@ def hello_world(args):
...
     # download the data
-    download_data(targets)
+    download_data(targets, args.download_proxy)
+
 
     # run it!
-    py_client(port_expose=args.port_expose, host=args.host).index(
-        input_fn(targets['index']['filename']), batch_size=args.index_batch_size)
+    py_client(port_expose=args.port_expose, host=args.host, batch_size=args.index_batch_size).index(
+        input_numpy(targets['index']['data']), batch_size=args.index_batch_size)
```
Comment (JoanFM) [inline helloworld-in-cs/client.py:34]: targets['index''] does not have `data`

Comment (YueLiu-jina) [reply helloworld-in-cs/client.py:34]: Yes, I also get very confused at this. When I print `targets['index']['data']` I also get a key error. But `targets['index']['data']` is how it works, and `targets['index']['filename']`

Comment (YueLiu-jina) [reply helloworld-in-cs/client.py:34]: This change to 'data' is inspired by our Jina repo's helloworld code.

Comment (YueLiu-jina) [reply helloworld-in-cs/client.py:34]: targets['index']['data'] is add in the `download_data(targets, args.download_proxy)` function in the previous step

**Pull request merged**